demisto_sdk/commands/common/content/objects/pack_objects/lists/lists.py

```diff
@@ -1,6 +1,7 @@
+import pprint
 from typing import Union
 
-import demisto_client
+from demisto_client.demisto_api import DefaultApi
```

Comment:
How do these imports look on other files?

Reply:
I think it's the right import, because we are specifically using it,
In addition, there is like this in `Job Upload`